Assets/MRTK/Core/Utilities/Editor/ScriptedImporterAssetReimporter.cs

```diff
@@ -41,12 +41,7 @@ public static void OnPostprocessAllAssets(string[] importedAssets, string[] dele
 
                         assetsAttemptedToReimport[asset] = ++numAttempts;
 
-                        if (numAttempts <= 3)
-                        {
-                            Debug.LogWarning($"Asset '{asset}' appears to have failed importing, will attempt to re-import. Attempt: {numAttempts}");
-                            AssetDatabase.ImportAsset(asset);
```

Comment:
Removing this line as well means that the re-import is never attempted

Reply:
good catch!

Reply:
I think just line 46 can be removed